test.js

```diff
@@ -65,23 +65,25 @@ test('match exact URLs', t => {
 });
 
 test('match URLs in text', t => {
-	const fixture = [
-		'Lorem ipsum //dolor.sit',
-		'<a href="http://example.com">example.com</a>',
-		'[and another](https://another.example.com)',
-		'Foo //bar.net/?q=Query with spaces'
-	].join('\n');
+	const fixture = `
+		Lorem ipsum //dolor.sit
+		<a href="http://example.com">example.com</a>
+		<a href="http://example.com/with-path">with path</a>
+		[and another](https://another.example.com) and
```

Comment:
I noticed that if you put a trailing `/` at the end of `https://another.example.com` the regex will pick up the trailing `)` as well. Can we exclude characters like `)([]<>` etc as well?